variables.tf

```diff
@@ -14,6 +14,24 @@ variable "instance_name" {
   description = "Name of the security and compliance instance that will be provisioned by this module"
 }
 
+variable "provider_type_instance_name" {
+  type        = string
+  description = "The name for the provider_type instance"
+  default     = null
+}
```

Comment:
I don't think we need to expose this, lets just hard code it to "workload-protection-instance"